packages/block-library/src/avatar/block.json

```diff
@@ -49,5 +49,18 @@
 		}
 	},
 	"editorStyle": "wp-block-avatar",
-	"style": "wp-block-avatar"
+	"style": "wp-block-avatar",
+	"__experimentalAutoInsert": {
+		"core/comment-template": {
+			"position": "lastChild",
+			"attrs": {
```

Comment:
why "attrs" and what is this exactly?

Reply:
After some reading, this looks like the block format we use temporarily after grammar parsing. I'm not sure I like the idea of making this format an official API in block.json, can't we use the final block object instead?